flow-typed/npm/babel-cli_vx.x.x.js

```diff
@@ -0,0 +1,108 @@
+// flow-typed signature: 8615cf6c39596ff7ba27c4a0377fe299
```

Comment:
That is part of the flow scheme, so that the consumer is in control of the rate of change of API evolution. There's a good chance that will change when flow reaches 1.0.